### ISSUE TITLE
aria-label issues 

### ISSUE THREAD

**Issue by jarieagito** [jarieagito/accessibility-fixer-upper#1]
Hi there,

I've been trying to figure out how to correctly use the aria-label in my index.html file for this assignment, it keeps telling me that I need a "unique aria-label" even when I followed the instructions from the accessibility video. Here's screenshots of my coding and the markbot issue

@mackenzierose
@muxlowc

<img width="1602" alt="Screen Shot 2020-04-01 at 8 14 21 PM" src="https://user-images.githubusercontent.com/55108790/78198064-6d1ed200-7455-11ea-9a1c-07604cebc33a.png">

**Comment by muxlowc** [comment]
@jarieagito hmm, it looks like you have two `<ol>`s , maybe try making it one `<ol>` with three `<li>`s? I feel like markbot might be weird because of that?

**Comment by MacKenzieRose** [comment]
Have you been able to solve this issue?

**Comment by jarieagito** [comment]
Yes! I just need to mess around with font sizes and paddings. I'm having trouble understanding why everything is in bold and it's asking for higher contrast which I'm unsure how to do.

Thanks so much for helping, stay safe!

Jarie Agito
________________________________

**Comment by muxlowc** [comment]
@jarieagito if you get your markbot screenshots down to about 14-15%, but can't figure out what else to do, let me know as i might need to give you the mark manually for this one!

**Comment by jarieagito** [comment]
@muxlowc I have a question about the SAAS website sketches, are we able to use the website idea we made in Jared's Interactive Design class? My website for that talks about Password Security.

**Comment by muxlowc** [comment]
@jarieagito Yes you can - I just ask that you mark it up showing where your html tags will go (ie. header/main/ sections / grids & units + sizing / footer etc etc)

**Comment by jarieagito** [comment]
Okay, and another question is can both tour.html and sign-up.html just be in the main page or does the nav links have to go to another site? 
@muxlowc

**Comment by muxlowc** [comment]
@jarieagito you need to have links to 3 separate pages.

**Comment by jarieagito** [comment]
I've attached my sketches for the SAAS websites, I haven't made the mobile versions for them. @muxlowc 
![agito-jarie-page-1](https://user-images.githubusercontent.com/55108790/78840335-72859a80-79c8-11ea-88ef-45bdceb2553a.jpg)
![agito-jarie-page-2](https://user-images.githubusercontent.com/55108790/78840337-744f5e00-79c8-11ea-82f6-383f99161c71.jpg)
![agito-jarie-page 3](https://user-images.githubusercontent.com/55108790/78840338-75808b00-79c8-11ea-8e85-75f0ea6bbd09.jpg)

**Comment by muxlowc** [comment]
@jarieagito looks great, thanks!

**Comment by MacKenzieRose** [comment]
If this issue has been addressed, please make sure to close it 😄
